src/CoreWCF.Primitives/src/CoreWCF/Channels/SslStreamSecurityUpgradeProvider.cs

```diff
@@ -36,7 +36,7 @@ private SslStreamSecurityUpgradeProvider(IDefaultCommunicationTimeouts timeouts,
             _clientCertificateAuthenticator = clientCertificateAuthenticator;
             IdentityVerifier = identityVerifier;
             Scheme = scheme;
-            SslProtocols = sslProtocols;
+            SslProtocols = SslProtocols.Default;
             ClientSecurityTokenManager = null; // Used for client but there's public api which need this and the compiler complains it's never assigned
```

Comment:
Revert change

Reply:
I know we discussed, I get this error => The specified value is not valid in the 'SslProtocolType' enumeration.\r\nParameter name: sslProtocolType... `   at System.Net.Security.SslState.ValidateCreateContext(Boolean isServer, String targetHost, SslProtocols enabledSslProtocols, X509Certificate serverCertificate, X509CertificateCollection clientCertificates, Boolean remoteCertRequired, Boolean checkCertRevocationStatus, Boolean checkCertName) in C:\Users\Administrator.MSCORE\AppData\Local\JetBrains\Shared\vAny\DecompilerCache\decompiler\BE5F9D07-12AA-4746-82F4-895E8CF62944\a4\c0fd1d78\SslState.cs:line 115
   at System.Net.Security.SslStream.BeginAuthenticateAsServer(X509Certificate serverCertificate, Boolean clientCertificateRequired, SslProtocols enabledSslProtocols, Boolean checkCertificateRevocation, AsyncCallback asyncCallback, Object asyncState) in C:\Users\Administrator.MSCORE\AppData\Local\JetBrains\Shared\vAny\DecompilerCache\decompiler\BE5F9D07-12AA-4746-82F4-895E8CF62944\62\9e0415cd\SslStream.cs:line 389
   at System.Net.Security.SslStream.<>c__DisplayClass35_0.<AuthenticateAsServerAsync>b__0(AsyncCallback callback, Object state) in C:\Users\Administrator.MSCORE\AppData\Local\JetBrains\Shared\vAny\DecompilerCache\decompiler\BE5F9D07-12AA-4746-82F4-895E8CF62944\62\9e0415cd\SslStream.cs:line 521
   at System.Threading.Tasks.TaskFactory`1.FromAsyncImpl(Func`3 beginMethod, Func`2 endFunction, Action`1 endAction, Object state, TaskCreationOptions creationOptions)
   at System.Net.Security.SslStream.AuthenticateAsServerAsync(X509Certificate serverCertificate, Boolean clientCertificateRequired, SslProtocols enabledSslProtocols, Boolean checkCertificateRevocation) in C:\Users\Administrator.MSCORE\AppData\Local\JetBrains\Shared\vAny\DecompilerCache\decompiler\BE5F9D07-12AA-4746-82F4-895E8CF62944\62\9e0415cd\SslStream.cs:line 521
   at CoreWCF.Channels.SslStreamSecurityUpgradeAcceptor.<OnAcceptUpgradeAsync>d__9.MoveNext() in C:\Users\Administrator\Projects\CoreWCF\src\CoreWCF.Primitives\src\CoreWCF\Channels\SslStreamSecurityUpgradeProvider.cs:line 261
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at CoreWCF.Channels.StreamSecurityUpgradeAcceptorBase.<AcceptUpgradeAsync>d__4.MoveNext() in C:\Users\Administrator\Projects\CoreWCF\src\CoreWCF.Primitives\src\CoreWCF\Channels\StreamSecurityUpgradeAcceptorBase.cs:line 28
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at CoreWCF.Channels.Framing.ServerFramingDuplexSessionMiddleware.<UpgradeConnectionAsync>d__5.MoveNext() in C:\Users\Administrator\Projects\CoreWCF\src\CoreWCF.NetTcp\src\CoreWCF\Channels\Framing\ServerFramingDuplexSessionMiddleware.cs:line 155
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at CoreWCF.Channels.Framing.ServerFramingDuplexSessionMiddleware.<OnConnectedAsync>d__2.MoveNext() in C:\Users\Administrator\Projects\CoreWCF\src\CoreWCF.NetTcp\src\CoreWCF\Channels\Framing\ServerFramingDuplexSessionMiddleware.cs:line 66`

Reply:
Can you provide instructions how you are getting this using your branch? None is definitely the right value, it's what we're using on .NET Framework and in WCF Core client.

Reply:
once you get the code from my branch GA-fix, just revert above line and run the test TransportWithMessageCredentialNetTCPTest